ddl/schema.go

```diff
@@ -70,6 +70,48 @@ func onCreateSchema(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	}
 }
 
+func rollbackDropSchema(t *meta.Meta, job *model.Job, dbInfo *model.DBInfo) (ver int64, err error) {
```

Comment:
Maybe we can only allow cancel `drop database` when `dbInfo.State` is public. This will make rollback process much simple. @winkyao @zimulala @ciscoxll PTAL. Current logic of `rollbackDropSchema` function is a little complexity。

Reply:
agree.